### PR TITLE
[#352] dashboardview (connects #352)

### PR DESCRIPTION
**Pull request by MacNew**
screenshot from 2018-01-30 17-05-38

### CODE REVIEW

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            </mat-card-content>
+    </mat-card>
+</div>
+<div class="dashboard-card-view-library">
+    <mat-card>
+            <mat-card-content>
+                <mat-grid-list cols="6" rowHeight="2:1">
+                        <mat-grid-tile class="backgroundcolor-purple">
+                          <mat-icon style="color:white;font:bold;" class="md-icon-36">event </mat-icon>
+
```
Comment (nirojdyola) [inline src/app/dashboard/dashboard.component.html]: Could you please remove extra line?

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                </mat-grid-list>         
+          </mat-card-content>          
+    </mat-card>
+</div>
+
+<div class="dashboard-card-view-library">
+    <mat-card>
+            <mat-card-content>
+                <mat-grid-list cols="6" rowHeight="2:1">
+                        <mat-grid-tile class="backgroundcolor-green">
```
Comment (nirojdyola) [inline src/app/dashboard/dashboard.component.html]: There are more spaces before <mat-grid-tile> . Please check it once .

File: src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    </mat-card>
+</div>
+
+
+<div class="dashboard-card-view-library">
+    <mat-card>
+            <mat-card-content>
+                <mat-grid-list cols="6" rowHeight="2:1">
+                        <mat-grid-tile class="backgroundcolor-orange">
+                                        <mat-icon style="color:white;font:bold;" class="md-icon-36">library_books </mat-icon>
```
Comment (nirojdyola) [inline src/app/dashboard/dashboard.component.html]: There are more spaces. Please check it once .

File: src/app/dashboard/dashboard.component.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        <mat-grid-tile>Title Second</mat-grid-tile>
+                        <mat-grid-tile class="backgroundcolor-gray">Title Third</mat-grid-tile>
+                        <mat-grid-tile >Title fourth</mat-grid-tile>
+                        <mat-grid-tile class="backgroundcolor-gray">Title fifth</mat-grid-tile>
+                        <mat-grid-tile>Title Sixth </mat-grid-tile>         
+                </mat-grid-list>
+        </mat-card-content>
+    </mat-card>
+</div>
+
```
Comment (nirojdyola) [inline src/app/dashboard/dashboard.component.html]: Why there are two extra lines?

File: src/app/dashboard/dashboard.component.html (hunk 1 above)
Comment (MacNew) [reply src/app/dashboard/dashboard.component.html]: Ok

File: src/app/dashboard/dashboard.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
+.dashboard-card-view {
```
Comment (razu9861) [inline src/app/dashboard/dashboard.scss]: Please follow indentation rule.

File: src/app/dashboard/dashboard.component.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
+<mat-card class="dashboard-card-view">
+     <mat-icon>home</mat-icon>
+          DashBoard
+</mat-card>
+<mat-divider></mat-divider>
+<div class="dashboard-card-view-library">
+    <mat-card>
+        <mat-card-content>
```
Comment (razu9861) [inline src/app/dashboard/dashboard.component.html]: indentaion there are extra spaces

File: src/app/dashboard/dashboard.scss [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+@mixin md-icon-size($size: 24px) {
+    font-size: $size;
+    height: $size;
+    width: $size;
+  }
+  .md-icon-36 {
+    @include md-icon-size(46px);
+  }
+
```
Comment (razu9861) [inline src/app/dashboard/dashboard.scss]: so many extra lines

File: src/app/dashboard/dashboard.component.ts [hunk 8]
```diff
@@ -4,9 +4,8 @@ import { UserService } from '../shared/user.service';
 
 // Main page once logged in.  At this stage is more of a placeholder.
 @Component({
-  template: `
-    <div id="greeting">Hi, {{name}}</div>
```
Comment (razu9861) [inline src/app/dashboard/dashboard.component.ts]: can we have greeting messages for now.

File: src/app/dashboard/dashboard.component.html (hunk 3 above)
Comment (razu9861) [inline src/app/dashboard/dashboard.component.html]: styles should go in scss file, bootstrap class should not be used

File: src/app/dashboard/dashboard.component.ts (hunk 8 above)
Comment (lmmrssa) [reply src/app/dashboard/dashboard.component.ts]: its ok not to have greeting message

File: src/app/dashboard/dashboard.component.html [hunk 9]
```diff
@@ -0,0 +1,74 @@
+<mat-card class="dashboard-card-view">
+  <div class="dashboard-contain-image-and-string">
```
Comment (lmmrssa) [inline src/app/dashboard/dashboard.component.html]: can we have i18n translation tag added in all places

**Comment by lmmrssa** [comment]
images used should be from the icon drive and also there is white space at bottom of all sections

**Comment by MacNew** [comment]
screenshot from 2018-02-01 11-53-05

**Comment by MacNew** [comment]
screenshot from 2018-02-02 16-27-14

**Comment by MacNew** [comment]
screenshot-2018-2-5 planet learning

File: src/app/dashboard/dashboard.component.ts [hunk 10]
```diff
@@ -4,19 +4,94 @@ import { UserService } from '../shared/user.service';
...
   constructor(
     private userService: UserService
   ) {}
 
   ngOnInit() {
     Object.assign(this, this.userService.get());
   }
+
+  nextLibrary() {
+    this.myLibrary = this.myLibrary.concat( this.myLibrary.splice( 0, 4 ) );
```
Comment (lmmrssa) [inline src/app/dashboard/dashboard.component.ts]: instead of playing around with array add pointer to point out the current position

**Comment by lmmrssa** [comment]
@paulbert shall we make content scrollable or sort the data instead to avoid scrolling half of the content of a card

**Comment by lmmrssa** [comment]
closing icon on favor of https://github.com/ole-vi/planet/pull/393

**Pull request closed without merging**